src/status_im/ui/components/large_toolbar/view.cljs

```diff
@@ -82,4 +82,26 @@
                                           (styles/minimized-toolbar-fade-out anim-opacity)
                                           #(reset! to-hide false)))))
     {:minimized-toolbar [minimized-toolbar header-in-toolbar nav-item toolbar-action-items anim-opacity]
-     :content-with-header [flat-list-with-large-header header content list-ref scroll-y]}))
\ No newline at end of file
+     :content-with-header [flat-list-with-large-header header content list-ref scroll-y]}))
+
+(defn add-listener [anim-opacity scroll-y]
+  (let [to-hide  (reagent/atom true)]
```

Comment:
`@to-hide` can be `false` by default, no need to run animation on initialisation

Reply:
also no need to use `reagent/atom` here, it should be just `atom`

Reply:
Will fix in next commit.